src/components/procedure/_pr-comments-in-sast.mdx

```diff
@@ -0,0 +1,24 @@
+## Configure comments for Semgrep Code
+
+<span>In addition to setting up the connection between Semgrep and {props.name}, you must assign rules to Comment or Block mode. This customization enables you to:</span>
+
+- <span>Manage the <strong>amount of {props.comment_type} comments</strong> your developers receive.</span>
+- Ensure that only rules that meet your criteria, such as high severity or high confidence rules, produce comments visible to developers, reducing noise.
+
+:::tip
+<span>Rules in <strong>Block mode</strong> prevent the {props.comment_type} from being merged without triage or remediation.</span>
+:::
+
+### Set rules to Comment or Block mode
+
+The following instructions let you customize what findings or security issues your developers see as comments in their PRs:
```

Comment:
final instance of `PRs` that should be `MRs` in the GitLab doc

Reply:
Fixed! Nice catch.